### ISSUE TITLE
Add mouse copy

### ISSUE THREAD

**Issue by stoney** [Clozure/ccl#319]
Trac issue #177 (Implement Mouse Copy) didn't show up here on github. I, and I'm sure many others, would like to see this implemented in Clozure CL. There is code from MCL that could be ported to CCL. Is there a reason why this remains unimplemented since it was requested 13 years ago?

**Comment by svspire** [comment]
Yes. Because it's hard. Much harder than it was in MCL. I've beat my head against this issue several times without success; the problem is that Cocoa gets the click event before Lisp does, which was not the case with Quickdraw. Somebody who knows Cocoa better than me will have to solve it.

**Comment by stoney** [comment]
I'm sorry that it's such a problem! I miss mouse copy.
